File: src/main/java/org/myfintech/payment/exception/Http404NotFoundException.java
package org.myfintech.payment.exception;

/**
 *
 * @author : Dhanuka Ranasinghe
 * @since : Date: 05/07/2025
 */
public class Http404NotFoundException extends RuntimeException
{

    private static final long serialVersionUID = 1L;

	public Http404NotFoundException( String message )
    {
        super( message );
    }

    public Http404NotFoundException( String message, Throwable cause )
    {
        super( message, cause );
    }

}

File: src/main/kotlin/org/myfintech/payment/exception/Http404NotFoundException.kt
package org.myfintech.payment.exception

/**
 *
 * @author : Dhanuka Ranasinghe
 * @since : Date: 05/07/2025
 */
class Http404NotFoundException : RuntimeException {
    constructor(message: String?) : super(message)

    constructor(message: String?, cause: Throwable?) : super(message, cause)

    companion object {
        private const val serialVersionUID = 1L
    }
}

